Dockerfile

```diff
@@ -1,14 +1,13 @@
 FROM alpine:3.13
 
-# Install packges needed
+# Install required packages
 RUN apk --no-cache add ca-certificates curl bash jq py3-pip && \
     pip install awscli
 
-COPY ecs-deploy /ecs-deploy
-RUN chmod a+x /ecs-deploy
+COPY ecs-deploy /usr/local/bin/ecs-deploy
+RUN ln -s /usr/local/bin/ecs-deploy /ecs-deploy
 
 COPY test.bats /test.bats
 COPY run-tests.sh /run-tests.sh
-RUN chmod a+x /run-tests.sh
 
-ENTRYPOINT ["/ecs-deploy"]
+ENTRYPOINT ["ecs-deploy"]
```

Comment:
I don't know, but I wonder if this might need to be "./ecs-deploy"

Reply:
I was thinking the symlink command was creating it in the current folder, but I see now that it was putting it in the root folder.

Reply:
The symlink is for backward comnpatibility in case people customized the execution of the container, in order to find the exec at the previous location.

Reply:
Is `/usr/local/bin` on the `PATH` by default?  Years ago it depended upon the logged in user (normal vs. root), the shell, the version of cron, the Unix distribution, etc.

Reply:
Yes, it is.

Reply:
Yes, it is.